src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/WorkflowRunTests.kt

```diff
@@ -454,6 +455,27 @@ class WorkflowRunTests : IntegrationTest()
         assertThat(errors[0]["code"].asText()).isEqualTo("bad-request")
     }
 
+    @Test
+    fun `can get workflow summary`()
+    {
+        val ref = getGitBranchCommit("master")
+        val sessionCookie = webRequestHelper.webLoginWithMontagu(runReportsPerm)
+        val response = webRequestHelper.requestWithSessionCookie(
+                "/workflows/summary",
+                sessionCookie,
+                ContentTypes.json,
+                HttpMethod.post,
+                """{"ref": "$ref", "reports": [{"name": "global", "params": {}}]}"""
+        )
+
+        assertSuccessful(response)
+        assertJsonContentType(response)
+        val json = JsonLoader.fromString(response.text)
+        val data =  json["data"].toString()
+        AssertionsForClassTypes.assertThat("missing_dependencies" in data)
```

Comment:
The above tests won't work as it won't let me cast a TextNode as an ArrayNode

Reply:
Sorry, that was really just an example of the sort of thing you could do, I couldn't remember what the expected data was exactly, of course missing dependencies is an object rather than an array. 

Actually, having run this myself, I can see that another reason it would be failing is that there is a bug - the response has two levels of our standard response format (the `{"data".., "errors", "status"..}` structure). You just need to remove `transform()` from the endpoint declaration in the route config - that's what's making it wrap whatever emerges from the controller in the response format, but it doesn't need to that here as the orderly server response has it already. Sorry, should have spotted that earlier. 

 Once I do that, this test code works for me:
```
        val json = JsonLoader.fromString(response.text)
        val data = json["data"]
        val reports = data["reports"] as ArrayNode
        assertThat(reports.count()).isEqualTo(1)
        assertThat((reports[0]["name"] as TextNode).textValue()).isEqualTo("global")
        assertThat((data["missing_dependencies"]["global"] as ArrayNode).count()).isEqualTo(0)
```